src/app/orbit-list/orbit-list.component.css

```diff
@@ -26,3 +26,6 @@ th, td {
 	margin: 5px 10px;
 	padding: 6px;
 }
+
+/* Zebra Stripes */
+tr:nth-child(even){ background-color:whitesmoke }
```

Comment:
Nice color combo!

Reply:
You could also apply a class to the `tr` where you use your `ngFor` and target it here vs. using `nth-child(even)` directly on the `tr` element itself to make it more specific.